### PR TITLE
Created list of the projects and delivery employee projects

### PR DESCRIPTION
**Pull request by jericoatos**

### CODE REVIEW

**Review by TaylorB-softdevkainos — APPROVED** [review]

File: src/main/java/org/example/controllers/DeliveryProjectController.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import org.example.services.DeliveryProjectService;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.sql.SQLException;
+
+public class DeliveryProjectController {
```
Comment (mlynch147) [inline src/main/java/org/example/controllers/DeliveryProjectController.java:11]: No default path set?

File: src/main/java/org/example/controllers/ProjectController.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.sql.SQLException;
+
+public class ProjectController {
```
Comment (mlynch147) [inline src/main/java/org/example/controllers/ProjectController.java:17]: No default path set?

File: src/main/java/org/example/controllers/ProjectController.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+    ProjectService projectService;
+
+    @GET
```
Comment (mlynch147) [inline src/main/java/org/example/controllers/ProjectController.java:21]: badly named - you are using a @GET but it looks like you are trying to do a PUT ???

File: src/main/java/org/example/models/Project.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    public double getValue() {
+        return value;
+    }
+
+    public void setValue(final double value) {
+        this.value = value;
+    }
+}
+
+
```
Comment (mlynch147) [inline src/main/java/org/example/models/Project.java:50]: delete the multiple blank lines

File: src/main/java/org/example/validators/DeliveryProjectValidator.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+import org.example.exceptions.InvalidException;
+
+import org.example.models.DeliveryProjectRequest;
+
+public class DeliveryProjectValidator {
+    public void validateDeliveryProject(
+            final DeliveryProjectRequest deliveryProjectRequest) throws
+            InvalidException {
+
```
Comment (mlynch147) [inline src/main/java/org/example/validators/DeliveryProjectValidator.java:11]: No method body?

**Review by orlaghmccloskey1 — APPROVED** [review]

**Pull request merged**